02_activities/assignments/assignment.sh

```diff
@@ -8,7 +8,7 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
-mk dir1 dir2 dir3 dir4 dir5
+mkdir dir1 dir2 dir3 dir4 dir5
 
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
 ls
```

Comment:
Because you have already merged into main, I can't highlight the exact code that you are wrong on.

Q3. Remove `cd dir2`, it is better practice to specify dir2 in the file path when you are creating them. Also add .txt extension to your file

Q4. This code will give you error. You should be appending "Hello world" into your file3, there shouldn't be any part where ls plays a role. Also missing .txt extension.

Q5. Missing file extension and specify dir2 in file path

Q6. Missing file extension and specify dir2 in file path